.circleci/config.yml

```diff
@@ -478,7 +478,7 @@ jobs:
             mkdir -p ~/.ssh
             ssh-keyscan github.com >> ~/.ssh/known_hosts
             git fetch --all
-            RELEASE_BRANCH=$(git branch -a --list *release/contracts/* | tail -1 | sed -e 's/remotes\/origin\///g')
+            RELEASE_BRANCH=8a8b82252e2da9a4afc743b1e27260d521729f74
```

Comment:
to me this seems a bit cryptic. With the deleted code you at least kind of get the gist of:  `*release/contracts/*`. (and this possibly isn't really a release anymore?). Might merit a code comment.

Reply:
Yeah, I added an echo to hopefully highlight this special case